components/ruby/README.md

```diff
@@ -81,10 +81,16 @@ ChefLicensing.configure do |config|
   config.chef_product_name = "chef"
   config.chef_executable_name = "inspec"
   config.chef_entitlement_id = "chef123"
+  config.chef_license_key="free-t0832est-1111-1te1-b1e5-011t182test3-111"
+  config.license_server_url_in_config_file = true
```

Comment:
maybe we can use `set_license_server_url_fron_config_file` or 'use_license_server_url_from_config_file`